testcases/OpTestInbandIPMI.py

```diff
@@ -73,6 +73,18 @@ def run_ipmi_cmd_on_host(self, i_cmd):
             raise OpTestError(l_msg)
         return l_res
 
+class BasicInbandIPMI(OpTestInbandIPMIBase):
+    ##
+    # @brief  It will check basic channel functionalities: info and authentication capabilities.
+    #
+    # @return l_res @type list: output of command or raise OpTestError
+    #
+    def test_channel(self):
+        print "Inband IPMI[OPEN]: Channel tests"
+        self.run_ipmi_cmd_on_host(BMC_CONST.IPMITOOL_OPEN + BMC_CONST.IPMI_CHANNEL_AUTHCAP)
+        self.run_ipmi_cmd_on_host(BMC_CONST.IPMITOOL_OPEN + BMC_CONST.IPMI_CHANNEL_INFO)
```

Comment:
For smoke test can we use some other command also apart from channel tests(It is giving some random output format).

Inband IPMI[OPEN]: Channel tests
ipmitool channel authcap 1 4; echo $?

Channel number             : 1
IPMI v1.5  auth types      : 
KG status                  : default (all zeroes)
Per message authentication : enabled
User level authentication  : enabled
Non-null user names exist  : yes
Null user names exist      : yes
Anonymous login enabled    : no
Channel supports IPMI v1.5 : no
Channel supports IPMI v2.0 : yes
0
1


ERROR

======================================================================
ERROR: test_channel (testcases.OpTestInbandIPMI.BasicInbandIPMI)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/pridhiviraj/new_optests/stewart_patch_series/op-test-framework/testcases/OpTestInbandIPMI.py", line 84, in test_channel
    self.run_ipmi_cmd_on_host(BMC_CONST.IPMITOOL_OPEN + BMC_CONST.IPMI_CHANNEL_AUTHCAP)
  File "/home/pridhiviraj/new_optests/stewart_patch_series/op-test-framework/testcases/OpTestInbandIPMI.py", line 71, in run_ipmi_cmd_on_host
    if int(l_res[-1]):
ValueError: invalid literal for int() with base 10: ''

----------------------------------------------------------------------
Ran 1 test in 58.686s

FAILED (errors=1)


May be we can use test_sensor_list to check for zero return code?